src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/ModelTests.kt

```diff
@@ -54,7 +69,45 @@ class ModelTests : DatabaseTest()
                         "id" to "modelId",
                         "description" to "description1",
                         "citation" to "Unknown citation",
-                        "modelling_group" to "groupId"
+                        "modelling_group" to "groupId",
+                        "gender_specific" to false,
+                        "gender" to "both",
+                        "current_version" to obj(
+                                "id" to 1,
+                                "version" to "v1",
+                                "note" to "Some note",
+                                "fingerprint" to "Some fingerprint",
+                                "is_dynamic" to true,
+                                "code" to "R",
+                                "max_countries" to 100,
+                                "model" to "modelId"
+                        )
+                )
+            })
+
+        }
+    }
+
+
+    @Test
+    fun `can get model without current version`()
```

Comment:
I'm also debating whether or not we should throw an `UnknownObjectError` in this case and/or exclude such models from the list coming back from `/models`. Certainly if a model is marked as current but has no current version, the modellers will not be able to upload estimates. But maybe for admin purposes it's better to see that the model is in an error state?

Reply:
Is that something that should be added to the admin screen then? I suppose it will be obvious from the missing metadata. Maybe it never actually happens in practice.

Reply:
I think lets not worry for now, this endpoint isn't even in use.